pcdsdevices/daq.py

```diff
@@ -572,7 +603,12 @@ def wait(self, timeout=None):
         def cb(*args, **kwargs):
             self._wait_done.set()
         self.finished_cb = cb
-        self._wait_done.wait(timeout=timeout)
+        finished = self._wait_done.wait(timeout=timeout)
```

Comment:
Is this method just a threaded version of `ophyd.status.wait`?

Reply:
I think you're on to something here: I didn't need to write this function at all. Not sure why I decided to implement it this way rather than just use the built-in wait. I also might need to change this when we update ophyd. I think I'll ignore it for now because it works and clean it up later.